libs/metrics/top_level_metrics.py

```diff
@@ -145,20 +145,12 @@ def calculate_or_copy_test_positivity(
     ) and has_data_in_past_10_days(data[CommonFields.NEGATIVE_TESTS])
     test_positivity = common_df.get_timeseries(data, CommonFields.TEST_POSITIVITY, EMPTY_TS)
     if positive_negative_recent or not test_positivity.notna().any():
-        cumulative_positive_tests = series_utils.interpolate_stalled_and_missing_values(
-            data[CommonFields.POSITIVE_TESTS]
-        )
-        cumulative_negative_tests = series_utils.interpolate_stalled_and_missing_values(
-            data[CommonFields.NEGATIVE_TESTS]
-        )
         method = _lookup_test_positivity_method(
             ts.provenance.get(CommonFields.POSITIVE_TESTS),
             ts.provenance.get(CommonFields.NEGATIVE_TESTS),
             log,
         )
-        test_positivity = calculate_test_positivity(
-            cumulative_positive_tests, cumulative_negative_tests
-        )
+        test_positivity = calculate_test_positivity(ts)
```

Comment:
small nit: don't love `ts` abbreviation, doesn't say much about the incoming data